### PR TITLE
add hipercard type, fix #58

### LINKED ISSUE #58: Hipercard type support
Based on this one https://github.com/moip/credit-card-validator/blob/4786855a/src/main/java/br/com/moip/creditcard/HipercardCreditCard.java#L6-L7

### PR DESCRIPTION
**Pull request by sibelius**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,7 +23,8 @@ var testOrder = [
   DISCOVER,
   JCB,
   UNIONPAY,
-  MAESTRO
+  MAESTRO,
+  HIPER_CARD
```
Comment (sibelius) [inline index.js]: how to define test_order?

**Comment by crookedneighbor** [comment]
You must provide documentation for this card type before it will be added.

File: index.js [hunk 2]
```diff
@@ -141,6 +144,26 @@ types[MAESTRO] = {
...
+  // 1) A validator from Moip:
+  // number != null && (
+  // 		number.matches("^606282[0-9]{10}$") || number.matches("^3841(0|4|6)0[0-9]{13}$")
+  // )
+  // https://github.com/moip/credit-card-validator/blob/4786855a/src/main/java/br/com/moip/creditcard/HipercardCreditCard.java#L6-L7
+  // 2) https://gist.github.com/fhferreira/3adc422e40bc31a39679
+  // 3) https://gist.github.com/danielfilho/9cd8fb61a39778be587c#bin-e-padrões-para-validação-de-cartão-de-crédito
+  exactPattern: /^(606282\d{10}(\d{3})?)|(3841(0|4|6)0\d{13})$/,
+  niceType: 'Hipercard',
+  type: HIPER_CARD,
```
Comment (crookedneighbor) [inline index.js]: Trailing comma.

File: index.js [hunk 3]
```diff
@@ -141,6 +144,26 @@ types[MAESTRO] = {
   }
 };
 
+types[HIPER_CARD] = {
+  code: {
+    name: CVC,
+    size: 3,
```
Comment (crookedneighbor) [inline index.js]: Trailing comma.

**Comment by sibelius** [comment]
I'll try to find the docs, tks

**Pull request closed without merging**